### PR TITLE
Add maximum number of stories limitation for selected stories block

### PR DESCRIPTION
**Pull request by ediamin**
This feature adds 20 stories limitation for selected stories block. This limitation can be override by the `web_stories_selected_stories_limit` filter.

### CODE REVIEW

File: includes/Block/Selected_Stories_Block.php [hunk 1]
```diff
@@ -61,6 +61,13 @@ class Selected_Stories_Block extends Latest_Stories_Block {
 	 */
 	protected $block_attributes = [];
 
+	/**
+	 * Maximum number of stories users can select
+	 *
+	 * @var int
+	 */
+	protected $max_num_of_stories = 0;
```
Comment (divyarajmasani) [inline includes/Block/Selected_Stories_Block.php:69]: you could set the default here `20`

File: includes/Block/Selected_Stories_Block.php [hunk 2]
```diff
@@ -69,6 +76,15 @@ class Selected_Stories_Block extends Latest_Stories_Block {
...
 	 */
 	public function init() {
+		/**
+		 * Filters the maximum number of stories users can select
+		 *
+		 * @since
+		 *
+		 * @param int $max_num_of_stories
+		 */
+		$this->max_num_of_stories = apply_filters( 'web_stories_selected_stories_limit', 20 );
```
Comment (divyarajmasani) [inline includes/Block/Selected_Stories_Block.php:86]: Can we not use the pre-defined class variable `$max_num_of_stories` instead of passing the `20`?

**Review by divyarajmasani — CHANGES_REQUESTED** [review]

File: includes/Block/Selected_Stories_Block.php (hunk 2 above)
Comment (ediamin) [reply includes/Block/Selected_Stories_Block.php:86]: Yes, we can. The only reason I initialized it with 0 and set 20 here is to use a filter. So do you prefer like this,
```php
...
protected $max_num_of_stories = 20;
...
...
...
$this->max_num_of_stories = apply_filters( 'web_stories_selected_stories_limit', $this->max_num_of_stories );

```

Should I update my code using this change?

Comment (divyarajmasani) [reply includes/Block/Selected_Stories_Block.php:86]: yes,

**Comment by ediamin** [comment]
@divyarajmasani Updated the code. Please review now.

**Review by divyarajmasani — APPROVED** [review]

**Pull request closed without merging**